serverless/cmd/integrate/main.go

```diff
@@ -18,30 +18,68 @@ package main
 
 import (
 	"flag"
+	"os"
 
 	"github.com/golang/glog"
+	"github.com/google/trillian-examples/serverless/api"
 	"github.com/google/trillian-examples/serverless/internal/log"
 	"github.com/google/trillian-examples/serverless/internal/storage/fs"
 	"github.com/google/trillian/merkle/rfc6962/hasher"
+	"golang.org/x/mod/sumdb/note"
 
 	fmtlog "github.com/google/trillian-examples/formats/log"
 )
 
 var (
 	storageDir = flag.String("storage_dir", "", "Root directory to store log data.")
+	initial    = flag.Bool("initial", false, "Set when creating a new log to initialise the structure.")
 )
 
 func main() {
 	flag.Parse()
 	h := hasher.DefaultHasher
 
+	privKey := os.Getenv("SERVERLESS_LOG_PRIVATE_KEY")
+	pubKey := os.Getenv("SERVERLESS_LOG_PUBLIC_KEY")
```

Comment:
Would  a `len() > 0` check on these be useful?

Reply:
Done.